x-pack/plugins/security_solution/public/flyout/document_details/right/components/investigation_guide.tsx

```diff
@@ -71,7 +76,12 @@ export const InvestigationGuide: React.FC = () => {
             { defaultMessage: 'investigation guide' }
           )}
         />
-      ) : !error && basicAlertData.ruleId && ruleNote ? (
+      ) : hasInvesigationGuide && isPreviewMode ? (
+        <FormattedMessage
+          id="xpack.securitySolution.flyout.right.investigation.investigationGuide.openFlyoutMessage"
+          defaultMessage="Open alert details flyout to view investigation guide."
```

Comment:
let's wait for @nastasha-solomon to make sure this text is what we want

Reply:
I guess if you want to merge this now and do a follow up PR to make the changes that'd be ok...